### PR TITLE
Add `include-compatible` config flag

### PR DESCRIPTION
**Pull request by Shatur**
When developing FPGA, code needs to be generated dynamically. Idiomatic way of doing it is using `include!` macro: https://doc.rust-lang.org/cargo/reference/build-script-examples.html#code-generation
But top-level attributes can't be used with it:
https://github.com/rust-lang/rfcs/issues/752
So I added a config flag that disables generation of all top-level attributes.

### CODE REVIEW

**Comment by burrbull** [comment]
I'm fine with adding flag, but I don't like flag name.

**Comment by Shatur** [comment]
What flag name would you suggest?

**Review by Emilgardis — DISMISSED** [review]

File: src/config.rs [hunk 1]
```diff
@@ -10,6 +10,7 @@ pub struct Config {
     pub atomics_feature: Option<String>,
     pub generic_mod: bool,
     pub make_mod: bool,
+    pub include_compatible: bool,
```
[the remark this answers is not in the document]
Comment (Shatur) [reply src/config.rs:13]: @Emilgardis how about `--skip-inner-attributes`?
Looks like they called "inner": https://doc.rust-lang.org/reference/attributes.html#attributes.

[the remark this answers is not in the document]
Comment (burrbull) [reply src/config.rs:13]: `--skip-crate-attributes`

[the remark this answers is not in the document]
Comment (Shatur) [reply src/config.rs]: I like it, done!

**Review by burrbull — APPROVED** [review]

**Pull request merged**